Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -101,7 +104,8 @@ BeamRelevant::BeamRelevant (std::string rd_name)
             ofs << "[13]pz_rms(kg*m/s)";  ofs << m_sep;
             ofs << "[14]gamma_rms()";     ofs << m_sep;
             ofs << "[15]emittance_x(m)";  ofs << m_sep;
-            ofs << "[16]emittance_z(m)";  ofs << std::endl;
+            ofs << "[16]emittance_x(m)";  ofs << m_sep;
```

Comment:
@ax3l @MaxThevenet Found a typo here, sorry I missed this when reviewing.
`ofs << "[16]emittance_x(m)";  ofs << m_sep;`
should be
`ofs << "[16]emittance_x(z)";  ofs << m_sep;`

Reply:
oh no! will do a follow-up PR!

Reply:
fix available in #900 :)